changes related to codespaces

KalpanaReddyC opened the pull request:
Below changes are done

1. Added .devcontainer/devcontainer.json file.
2. Renamed Client_Id, Client_Secret to PAYPAL_CLIENT_ID,PAYPAL_CLIENT_SECRET.
3. Updated package.json to include "kill-port" package.

KalpanaReddyC committed:
changes related to codespaces

sdarshale reviewed (changes requested):
- missing newline at the end of some files
- devcontainer file can use `--yes` flag to auto-install `kill-port`
- no package json changes required for `kill-port`. this can be fixed with `--yes` flag in devcontainer file
-  asismohanty - for client id and secret changes, please check with domain teams. changing existing code might have negative impact on existing set up.
- need to test `Open In Codespaces` link. it should show options for user inputs on client id secret etc.

KalpanaReddyC committed:
Reverting back PAYPAL_ changes and package file

KalpanaReddyC commented:
1. missing newline at the end of some files
     - Did not find any file, please specify the filename
2. devcontainer file can use --yes flag to auto-install kill-port
     - I have corrected it by adding "CI=true npx kill-port --yes " in devcontainer.json file    
3. no package json changes required for kill-port. this can be fixed with --yes flag in devcontainer file.
     - The above comment is applicable for this also.
4. asismohanty - for client id and secret changes, please check with domain teams. changing existing code might have negative impact on existing set up.
     - This change was asked on slack and so we did that. Reverted back those changes and committed the code.
5. need to test Open In Codespaces link. it should show options for user inputs on client id secret etc.
     - The link in Readme.md file seems to be working perfectly and able to enter values for client-Id and client-secret, if already those values are added then it will not prompt next time.

KalpanaReddyC committed:
Updated documentation Url(s) in devcontainer file

sdarshale reviewed (approved):
LGTM

devchristina commented:
LGTM
